README.md

```diff
@@ -120,30 +120,27 @@ converted into a flow of learning goals and questions. Note, again, that it is p
 building your actual lesson, to re-order these files, or omit one or more of them.
 
 * Prelude
-    * User profiles (academic and/or commercial) of cluster users
+    * [User profiles](
+      https://github.com/carpentries-incubator/hpc-intro/blob/gh-pages/_extras/learner-profiles.md)
```

Comment:
This link was added but the corresponding file was also removed in the same PR. I'll open a PR to reinstate the file and we can decide exactly what to do with it. Maybe this link just needs an update?

Reply:
Oops :laughing:

Reply:
I think we should update the link, rather than reinstating the file.

Reply:
There's no real good way of doing that because of the auto-rendering of markdown, best I can do is https://github.com/carpentries-incubator/hpc-intro/blame/gh-pages/_episodes/11-hpc-intro.md#L23

I'll open a PR that adds a heading to that section so I can link to that instead

Reply:
A couple of options:

1. Don't make it a direct link, as the profiles are the first thing in the lesson (with no lead-in... something else to fix).
2. Refer to the Learner Prerequisites (#308) instead of the lesson content, as that is actually "prelude" material.
2. Drop the "prelude" item, and move the mention of learner profiles (without linking) to the next list item, where it now belongs.